pkg/operation/botanist/controlplane.go

```diff
@@ -491,16 +523,26 @@ func (b *Botanist) DeploySeedLogging() error {
 
 	ct := b.Shoot.Info.CreationTimestamp.Time
 
+	sgFluentdSecret, err = b.K8sSeedClient.GetSecret(common.GardenNamespace, "fluentd-es-sg-credentials")
+	if err != nil {
+		return err
+	}
+
+	sgFluentdPasswordHash = string(sgFluentdSecret.Data["hashedPassword"])
```

Comment:
```suggestion
	sgFluentdPasswordHash = string(sgFluentdSecret.Data["bcryptHashedPassword"])
```
or something similar